test/e2e/node/pod_resize.go

```diff
@@ -170,7 +170,8 @@ func doPodResizeAdmissionPluginsTests() {
 			ginkgo.By("patching pod for resize with memory exceeding resource quota")
 			_, pErrExceedMemory := f.ClientSet.CoreV1().Pods(resizedPod.Namespace).Patch(ctx,
 				resizedPod.Name, types.StrategicMergePatchType, []byte(patchStringExceedMemory), metav1.PatchOptions{}, "resize")
-			gomega.Expect(pErrExceedMemory).To(gomega.HaveOccurred(), tc.wantMemoryError)
+			gomega.Expect(pErrExceedMemory).To(gomega.HaveOccurred())
+			gomega.Expect(pErrExceedMemory.Error()).To(gomega.ContainSubstring(tc.wantMemoryError))
```

Comment:
nit: prefer
```suggestion
			gomega.Expect(pErrExceedMemory).To(gomega.MatchError(gomega.ContainSubstring(tc.wantMemoryError)))
```

Same below